platform/core/src/services/MeasurementService/MeasurementService.js

```diff
@@ -184,6 +187,24 @@ class MeasurementService {
     return measurement;
   }
 
+  setMeasurementSelected(measurementUID, selected) {
+    const measurement = this.getMeasurement(measurementUID);
+    if (measurement) {
```

Comment:
Not a big deal but I useually like to have a flat code. So I would do it 

```
if (!measurement){
  return
}

// the rest of logic (less indentation)
```

Reply:
Sure.